rest-api/src/test/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1Test.java

```diff
@@ -62,6 +63,17 @@ void testGetFileById() throws IOException {
 				.isEqualTo("1234");
 	}
 
+	@Test
+	void testMarkFileAsProcessed() {
+		when(cpcFileService.processFileById(anyString())).thenReturn("success!");
+
+		ResponseEntity<String> response = cpcFileControllerV1.markFileProcessed("meep");
+
+		verify(cpcFileService, times(1)).processFileById(anyString());
+
+		assertThat(response.getBody()).isEqualTo("success!");
```

Comment:
We should also assert that the HTTP `OK` was returned too.

Reply:
Agreed. But I think I'll write a second test to test it separately. I kinda like the one assert per test.